gpii/node_modules/flowManager/src/SystemUtils.js

```diff
@@ -25,3 +26,33 @@ gpii.reservedGpiiKeys = ["noUser", "restore"];
 gpii.isReservedGpiiKey = function (gpiiKey) {
     return gpii.reservedGpiiKeys.indexOf(gpiiKey) !== -1;
 };
+
+// Solve GPII-3310 by ensuring fluid.log output in the cloud lies on a single line
+gpii.renderCloudLoggingArgs = function (args) {
```

Comment:
Shall a test be added for either this function or "gpii.withCloudLogging" grade?

Reply:
We will get regular feedback from the devops team about whether these logs meet their requirements and I think that is a better route than verifying some epiphenomena of the particular formatting in a test.